### ISSUE TITLE
Unable to access UI or API via NodePort in dev mode

### ISSUE THREAD

**Issue by jamesmmchugh** [hashicorp/vault-helm#451]
I'm installing the chart with the following values:
```
server:
  dev:
    enabled: true
    # Set VAULT_DEV_ROOT_TOKEN_ID value
    devRootToken: "something"
  service:
    enabled: true
    type: "NodePort"
ui:
  enabled: true
  serviceType: "NodePort"
```
However I always get 'connection refused' when I try and connect to either the UI or the API.
i.e.
```
export VAULT_ADDR='http://{myNode}:{servceNodePort}'
vault status
Error checking seal status: Get "http://{myNode}:{servceNodePort}/v1/sys/seal-status": dial tcp {nodeIP}:{serviceNodePort}: connect: connection refused
```
Other services exposed as NodePorts and in the cluster all work as expected.

**Comment by tvoran** [comment]
Hi @jamesmmchugh this is probably due to a bug that was fixed in #446. We'll probably be releasing it in the next couple weeks.

In the meantime, you can try setting VAULT_DEV_LISTEN_ADDRESS to "[::]:8200" to work around it.

```yaml
server:
  extraEnvironmentVars:
    VAULT_DEV_LISTEN_ADDRESS: "[::]:8200"
```

**Comment by tuteng** [comment]
I have the same problem

**Comment by jamesmmchugh** [comment]
> Hi @jamesmmchugh this is probably due to a bug that was fixed in #446. We'll probably be releasing it in the next couple weeks.
> 
> In the meantime, you can try setting VAULT_DEV_LISTEN_ADDRESS to "[::]:8200" to work around it.
> 
> ```yaml
> server:
>   extraEnvironmentVars:
>     VAULT_DEV_LISTEN_ADDRESS: "[::]:8200"
> ```

This worked for me in the meantime. Thanks.

**Comment by tvoran** [comment]
@jamesmmchugh Glad to hear it worked! I'll go ahead and close this issue for now.

@tuteng If the workaround isn't helping your issue, I'd suggest asking a question in our [discuss forum](https://discuss.hashicorp.com/c/vault) to hit a wider audience.
